src/app/components/tabs/index.html

```diff
@@ -1,15 +1,292 @@
+			</div>
+			<div class="two-col-image">				
+				<div class="screenshot screenshot-success">
+					<img class="bordered" src="~/assets/img/guidelines/tab/use-tasks.png" />
+					<div class="caption sky-text-success">Do use tabs to group content for different tasks.</div>
+				</div>
+			</div>
+		</div>
+		
+	<h3>Don't use when</h3>
+		<div class="two-col-right-image  sky-margin-stacked-content-section">
+			<div class="two-col-text">
+				<p>You already have a set of tabs on the page. Try changing the workflow rather than using multiple sets of tabs.</p>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot screenshot-danger sky-margin-stacked-content-section">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-double-tab-horz.png" />
+					<div class="caption sky-text-danger">Don't use multiple sets of tabs on a single page.</div>
+				</div>
+				<div class="screenshot screenshot-danger">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-double-tab-vert.png" />
+					<div class="caption sky-text-danger">Don't combine vertical tabs with horizontal tabs.</div>
+				</div>
+			</div>
+		</div>
+    
+    <div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<p>The tabs are likely to overflow the available horizontal space, which will lead to the collapsed tab dropdown and a large amount of unused space in the tab bar. Use <a routerLink="/components/vertical-tabs">vertical tabs</a> instead.</p>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot screenshot-danger">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-overflow.png" />
+					<div class="caption sky-text-danger">Don't use horizontal tabs when they are likely to overflow.</div>
+				</div>
+			</div>
+		</div>
+		
+	</sky-demo-page-content>
+	
+	<sky-demo-page-content sectionHeading="Anatomy">
+		<div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<div class="anatomy-list">
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">1</div><div class="anatomy-description">Selected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">2</div><div class="anatomy-description">Unselected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">3</div><div class="anatomy-description">Tab border</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">4</div><div class="anatomy-description">Tab count (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">5</div><div class="anatomy-description">Close tab button (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">6</div><div class="anatomy-description">Open tab button (optional)</div></div>
```

Comment:
I think we are missing an item here. We have the Open and Close buttons, but not the Add button. That's a separate button, right? I see it in the image below for the Open and close section and I see a property for it in the properties for the components, so I believe so.

Reply:
On the UX side we want to get away from having separate Add and Open buttons, since they are not being used consistently anyway. The desired interaction is to only use the Add button, with action buttons on the new tab view as to what to put into that view.

Reply:
A couple things:

First, the Anatomy list and graphic are out of whack. The label for item 6 is "Open tab button," but the graphic has the Add tab button.

Second, recommending only using the Add tab button seems confusing to me. Based on the demos, the Add button creates a new tab in the tabset, while the Open button (seems to) to open an existing tab that was closed. So I don't follow how the Add button is a replacement for the Open button or vice versa. If we are recommending only using the Add button, then how do users reopen tabs after they close them?

Finally, if this is correct and we are recommending that designers only use the Add button, then shouldn't we adjust the description of the `openTab` event that creates the Open button to let them know not to use it? And shouldn't we also add something to the design docs to call out that designers should not use the Open button? Just excluding the button from the Anatomy list and graphic doesn't seem sufficient to make it clear that we want users to use the Add tab button and not use the Open tab button.

Reply:
Updated the UX guidelines to refer to Add rather than Open. The expected UX for opening a closed tab is in the "Options > Add and close" section. We should update the demo to remove the Open button. Not sure what our deprecation plan is, whether we should deprecate the open button or leave it but recommend against using it.